### PR TITLE
[4.0] More state => stage renaming

### PR DESCRIPTION
**Pull request by bembelimen**
### Summary of Changes
Renaming the filter options to fit the new naming.

### Testing Instructions
Go to the article (or featured article) manager and check out the filter options

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_content.ini [hunk 1]
```diff
@@ -146,7 +146,8 @@ COM_CONTENT_SHOW_IMAGES_URLS_BACK_LABEL="Administrator Images and Links"
 COM_CONTENT_SHOW_IMAGES_URLS_FRONT_LABEL="Frontend Images and Links"
 COM_CONTENT_SHOW_PUBLISHING_OPTIONS_LABEL="Publishing Options"
 COM_CONTENT_SLIDER_EDITOR_CONFIG="Configure Edit Screen"
-COM_CONTENT_STATE="State"
+COM_CONTENT_STAGES="Stage"
+COM_CONTENT_CONDITIONS="Conditions"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_content.ini]: For consistency please change this to the singular

**Review by brianteeman — APPROVED** [review]
approve language string changes

File: administrator/language/en-GB/en-GB.com_content.ini (hunk 1 above)
Comment (bembelimen) [reply administrator/language/en-GB/en-GB.com_content.ini]: Thx

**Comment by wilsonge** [comment]
Conflicts here please

File: administrator/components/com_content/tmpl/articles/default.php [hunk 2]
```diff
@@ -110,7 +110,7 @@
 									<?php echo HTMLHelper::_('grid.checkall'); ?>
 								</td>
 								<th scope="col" style="width:1%" class="nowrap text-center">
-									<?php echo HTMLHelper::_('searchtools.sort', 'JSTATUS', 'a.state', $listDirn, $listOrder); ?>
+									<?php echo HTMLHelper::_('searchtools.sort', 'COM_CONTENT_STAGE', 'a.state', $listDirn, $listOrder); ?>
```
Comment (ggppdk) [inline administrator/components/com_content/tmpl/articles/default.php]: This looks really wrong
The column name should rename status, column currently includes 3 things
- Featured flag
- Condition (Icon)
- Workflow stage (Textual)

Comment (ggppdk) [reply administrator/components/com_content/tmpl/articles/default.php]: Please see my comment here:
https://github.com/joomla/joomla-cms/issues/21531#issuecomment-414113171

Comment (bembelimen) [reply administrator/components/com_content/tmpl/articles/default.php]: Yeah, it's a bit tricky here (somewhere is an open issue for this) I think we can change it then. I reverted it, thx.

**Comment by ggppdk** [comment]
I have tested this item :white_check_mark: successfully on 83cd4220687670f4e93d8c629fba54de6bd1ec78<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21662">issues.joomla.org/tracker/joomla-cms/21662</a>.</sub>

File: administrator/language/en-GB/en-GB.com_content.ini [hunk 3]
```diff
@@ -146,7 +146,8 @@ COM_CONTENT_SHOW_IMAGES_URLS_BACK_LABEL="Administrator Images and Links"
 COM_CONTENT_SHOW_IMAGES_URLS_FRONT_LABEL="Frontend Images and Links"
 COM_CONTENT_SHOW_PUBLISHING_OPTIONS_LABEL="Publishing Options"
 COM_CONTENT_SLIDER_EDITOR_CONFIG="Configure Edit Screen"
-COM_CONTENT_STATE="State"
+COM_CONTENT_STAGE="Stage"
+COM_CONTENT_CONDITION="Conditions"
```
Comment (ggppdk) [inline administrator/language/en-GB/en-GB.com_content.ini]: Is this ok ? that the constant is COM_CONTENT_CONDITION but the translation is plural ?

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 83cd4220687670f4e93d8c629fba54de6bd1ec78<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21662">issues.joomla.org/tracker/joomla-cms/21662</a>.</sub>

File: administrator/language/en-GB/en-GB.com_content.ini (hunk 3 above)
Comment (bembelimen) [reply administrator/language/en-GB/en-GB.com_content.ini]: It makes no sense to have this constants at all. Thx.

File: administrator/components/com_content/forms/filter_featured.xml [hunk 4]
```diff
@@ -12,7 +12,6 @@
 		<field
 				name="stage"
```
Comment (brianteeman) [inline administrator/components/com_content/forms/filter_featured.xml]: This block has been indented by one too many tabs

File: administrator/components/com_content/forms/filter_featured.xml [hunk 5]
```diff
@@ -22,7 +21,6 @@
 		<field
 				name="condition"
 				type="workflowcondition"
-				label="COM_CONTENT_CONDITION"
```
Comment (brianteeman) [inline administrator/components/com_content/forms/filter_featured.xml]: This block has been indented by one too many tabs

File: administrator/components/com_content/forms/filter_featured.xml (hunk 4 above)
Comment (bembelimen) [reply administrator/components/com_content/forms/filter_featured.xml]: Thy

File: administrator/components/com_content/forms/filter_featured.xml [hunk 6]
```diff
@@ -10,19 +10,19 @@
...
 		<field
-				name="stage"
-				type="workflowstage"
-				onchange="this.form.submit();"
-				activeonly="true"
+			name="stage"
+			type="workflowstage"
+			onchange="this.form.submit();"
+			activeonly="true"
 		>
```
Comment (brianteeman) [inline administrator/components/com_content/forms/filter_featured.xml]: and this one should be indented one more - sorry didnt spot it before

and same for the other block

File: administrator/components/com_content/tmpl/featured/default.php [hunk 7]
```diff
@@ -96,7 +96,7 @@
 									<?php echo HTMLHelper::_('grid.checkall'); ?>
 								</td>
 								<th scope="col" style="width:1%; min-width:85px" class="nowrap text-center">
-									<?php echo JText::_("COM_CONTENT_STATE") ?>
+									<?php echo JText::_("JSTATUS") ?>
```
Comment (Quy) [inline administrator/components/com_content/tmpl/featured/default.php]: Add semicolon

File: administrator/components/com_content/forms/filter_articles.xml [hunk 8]
```diff
@@ -10,20 +10,17 @@
...
 		<field
-			name="state"
+			name="stage"
 			type="workflowstage"
-			label="COM_CONTENT_STATES"
 			onchange="this.form.submit();"
 			activeonly="true"
 			>
-			<option value="">COM_CONTENT_SELECT_STATE</option>
+			<option value="">COM_CONTENT_SELECT_STAGE</option>
```
Comment (infograf768) [inline administrator/components/com_content/forms/filter_articles.xml]: i suggest to create and rather use aJOPTION_SELECT_STAGE in en-GB.ini as we do that for all select.

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Pull request merged**